Mathlib/Dynamics/Ergodic/Action/OfMinimal.lean

```diff
@@ -0,0 +1,125 @@
+/-
+Copyright (c) 2024 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import Mathlib.Dynamics.Ergodic.Action.Regular
+import Mathlib.MeasureTheory.Measure.ContinuousPreimage
+import Mathlib.MeasureTheory.Measure.Haar.Unique
+
+/-!
+# Ergodicity from minimality
+
+
+-/
+
+open MeasureTheory Filter Set Function
+open scoped Pointwise
+
+variable {X : Type*} [TopologicalSpace X] [R1Space X] [MeasurableSpace X] [BorelSpace X]
```

Comment:
@sgouezel Could you please have a look at the theorems in this file? Do I use the right assumptions, or some of them aren't needed?

I'm going to add docstrings and move more assumptions to `variable`s once API is stable.

Reply:
I reorganized to use `section`s and `variable`s. I'll add more docstrings tomorrow.